### ISSUE TITLE
large image blocked

### ISSUE THREAD

**Issue by malwcal** [apparition47/MailTrackerBlocker#88]
In the emails sent from my local newspaper, the large image at the top of each post is blocked.

**Comment by apparition47** [comment]
Are you on the latest `0.3.24`? There was a change to fix a similar issue (#75). If you're still seeing this issue on the latest, could you provide the raw email `eml` file (File > Save As > Raw Message Source)?

**Comment by malwcal** [comment]
[brokennews.zip](https://github.com/apparition47/MailTrackerBlocker/files/6178839/brokennews.zip)
.eml file with broken image

**Comment by apparition47** [comment]
Maybe I'm terrible at `Where's Waldo?` but I'm struggling to see where the issue is (output looks the same as the [website too](https://view.e.ntnews.com.au/?qs=a7843c5d3ecb692a1d670518102f049a9bf46d24c17674a383ed96b3901c2b73ad0ee2b15acd675482ed64689308c422712ead0110d29fb53cad724f0be106e720bcaf270b98edc75d3784e885b5120f48aca891c9661a7d)). Could you circle the missing image? Try the latest version perhaps, if you're not on it already?

<img width="400" alt="Screen Shot 2021-03-22 at 10 22 42 AM" src="https://user-images.githubusercontent.com/3298414/111929018-95855d80-8af8-11eb-8274-07bb6d95f049.png">

**Comment by malwcal** [comment]
Ok, hangon, its probably the version.  I am on 0.3.21 and the image of the politicians head underneath "new update" is missing.

But I am now stuck trying to upgrade the plugin.

`brew upgrade apparition47/tap/mailtrackerblocker
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/cask).
==> Updated Casks
Updated 1 cask.

==> Upgrading 1 outdated package:
apparition47/tap/mailtrackerblocker 0.3.21 -> 0.3.24
==> Upgrading mailtrackerblocker
==> Downloading https://github.com/apparition47/MailTrackerBlocker/releases/download/0.3.24/MailTrackerBlocker.pkg
Already downloaded: /Users/malcolm/Library/Caches/Homebrew/downloads/ccd4e5f426d507eae5ecb1826de00922f6702fa4a793f5d0de5390091c4b09ce--MailTrackerBlocker.pkg
==> Uninstalling packages:
com.onefatgiraffe.mailtrackerblocker
find: /private/tmp/MailTrackerBlocker-Installation-Temp: No such file or directory
==> Purging files for version 0.3.24 of Cask mailtrackerblocker
Error: apparition47/tap/mailtrackerblocker: Failure while executing; `/usr/bin/sudo -E -- /usr/bin/xargs -0 -- /bin/bash -c set\ -euo\ pipefail'
''
'for\ path\ in\ \"\$\{@\}\"\;\ do'
'\ \ if\ \[\[\ -e\ \"\$\{path\}/.DS_Store\"\ \]\]\;\ then'
'\ \ \ \ /bin/rm\ -f\ \"\$\{path\}/.DS_Store\"'
'\ \ fi'
''
'\ \ \#\ Some\ packages\ leave\ broken\ symlinks\ around\;\ we\ clean\ them\ out\ before'
'\ \ \#\ attempting\ to\ \`rmdir\`\ to\ prevent\ extra\ cruft\ from\ accumulating.'
'\ \ /usr/bin/find\ \"\$\{path\}\"\ -mindepth\ 1\ -maxdepth\ 1\ -type\ l\ \!\ -exec\ /bin/test\ -e\ \{\}\ \\\;\ -delete'
''
'\ \ if\ \[\[\ -L\ \"\$\{path\}\"\ \]\]\;\ then'
'\ \ \ \ \#\ Delete\ directory\ symlink.'
'\ \ \ \ /bin/rm\ \"\$\{path\}\"'
'\ \ elif\ \[\[\ -d\ \"\$\{path\}\"\ \]\]\;\ then'
'\ \ \ \ \#\ Delete\ directory\ if\ empty.'
'\ \ \ \ /usr/bin/find\ \"\$\{path\}\"\ -maxdepth\ 0\ -type\ d\ -empty\ -exec\ /bin/rmdir\ \{\}\ \\\;'
'\ \ else'
'\ \ \ \ \#\ Try\ \`rmdir\`\ anyways\ to\ show\ a\ proper\ error.'
'\ \ \ \ /bin/rmdir\ \"\$\{path\}\"'
'\ \ fi'
'done'
' --` exited with 1. Here's the output:
find: /private/tmp/MailTrackerBlocker-Installation-Temp: No such file or directory
`

**Comment by apparition47** [comment]
Pre-0.3.22 doesn't play well with the Brew uninstallation scripts but it's been fixed. You'll need to see the comment at the bottom here for a fix (also see the official Cask migration comment below as well): apparition47/homebrew-tap#1 .

**Comment by malwcal** [comment]
Lol, I hadn't read that comment, but when your last post came in just as I was about to press enter on `mkdir /private/tmp/MailTrackerBlocker-Installation-Temp` as it seems a good way to fix the problem with the find failing.

I gave some bad information: the image that is broken for me is the banner that says NT News.

<img width="721" alt="image" src="https://user-images.githubusercontent.com/1244623/111929871-1f382980-8aff-11eb-983a-b9ccb14b5028.png">

**Comment by apparition47** [comment]
Odd. I'm seeing the header on my setup.

1. Did Mail restart properly during installation?
2. Can you confirm you see the [header image here](https://s1.wp.com/wp-content/themes/vip/newscorpau-tangram-ui/src/elements/tge-newsletter-site-logo/img/ntnews.png)? 
3. Do you see the header even with the plugin disabled?

**Comment by malwcal** [comment]
Yes,

There is something strange going on, but I don't think its the plugin.  I have restarted without the plugin and the image still seems broken.  

Again we are in lock step: I just tried all that and also checked that I cannot view that image in safari, nor download it with curl.  Which is odd.  I also tried from my home machine on a completely different network and it also cannot download that image.

But when I view the email in gmail's web interface it is there.

Some kind of CDN issue I suspect.  We can close this issue.
